pyproject.toml

```diff
@@ -84,6 +84,13 @@ select = [
     "UP",   # pyupgrade
     "W",    # pycodestyle warnings
 ]
+extend-ignore = [
+    "ANN002", # Missing type annotation for *args
+    "ANN003", # Missing type annotation for **kwargs
+    "D203", # Contradicts D211 and gives warning
+    "D213", # Contradicts D212 and gives warning  
```

Comment:
We can also ignore [D413](https://docs.astral.sh/ruff/rules/missing-blank-line-after-last-section/)